apps/common/src/python/mediawords/languages/lt/lt_stop_words.txt

```diff
@@ -3,131 +3,481 @@
 #
 # Sources:
 #     http://www.filewatcher.com/p/punbb-1.2.16.tbz.620109/www/punbb/lang/Lithuanian/stopwords.txt.html
+#     https://github.com/stopwords-iso/stopwords-lt/blob/master/stopwords-lt.txt
 #     auto-generated sources
 #
 
-a
+abi
```

Comment:
The Lithuanian stopword list is not particularly important for us as no Lithuanians do research with Media Cloud (some did try it in the past though, and one friend of mine even defended her Master's using the tool), but in this particular case both the old stopword list and the new one should be joined together as both include very valid, legit, one-size-fits-all stopwords.

Reply:
Very cool, thank you very much!